src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -137,6 +139,31 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchCurrentUser(currentUser);
     }
 
+    componentDidUpdate({ annotatorState: prevAnnotatorState }: Props): void {
+        const { error: prevError } = prevAnnotatorState;
+        const {
+            annotatorState: { error },
+        } = this.props;
+
+        if (!prevError && error && error.code === ERROR_CODE_FETCH_ANNOTATIONS) {
```

Comment:
Should this just check that `prevError !== error` instead of `!prevError && error` in case there was previously another error such as annotation create?